services/theme_service.py

```
@@ -0,0 +1,207 @@
+        except Exception as exc:
+            raise ThemeError(f"持久化活跃主题失败: {exc}") from exc
+
+        return {"name": name, "active": True}
+
+    def get_active_theme(self) -> str | None:
+        """获取当前持久化的活跃主题名称。"""
+        if self.settings_service is None:
+            return None
+        try:
+            settings = self.settings_service.get_settings()
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)